backend/app/controllers/spree/admin/products_controller.rb

```diff
@@ -25,6 +25,11 @@ def update
         if params[:product][:option_type_ids].present?
           params[:product][:option_type_ids] = params[:product][:option_type_ids].split(',')
         end
+        if updating_variant_property_rules?
+          params[:product][:variant_property_rules_attributes].each do |index, param_attrs|
+            param_attrs[:option_value_ids] = param_attrs[:option_value_ids].split
```

Comment:
It is, the value set on the view is an array of integers (I don't do a `.join(',')` on the values) so it's space delimited when coming back.